ToolBox/CoinApiDataConverter/CoinApiDataConverter.cs

```diff
@@ -83,6 +83,7 @@ public bool Run()
             var success = true;
 
             var fileToProcess = _rawDataFolder.EnumerateFiles("*.gz")
+                .Where(f => f.Name.Split('_').Length == 4)
```

Comment:
here as well -- _although_ -- the commit message is **terrific** so the code comment isn't mandatory 👍